Epic 3 - ReportApp/ReportApp/Models/CurrentShopItems/CurrentShopItemsConfiguration.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Text.Json;
+
+namespace ReportApp.Models.CurrentShopItems;
+
+public class CurrentShopItemsConfiguration
```

Comment:
Why we don't inherit it from IReportConfiguration? And please, look for common configuration properties in all your configs, maybe we can extract them?